embuilder.py

```diff
@@ -91,12 +90,17 @@ def get_help():
   return '''
 Available targets:
 
-  build %s
+  build / clear %s
 
-Issuing 'embuilder.py build ALL' causes each task to be built.
+Issuing 'embuilder build ALL' causes each task to be built.
+Issuing 'embuilder clear' cases the entire cache to be cleared.
```

Comment:
perhaps this should also use `ALL` for consistency with `BUILD` so it's symmetrical?

Reply:
Actually I did not implement that part yet, I will remove that line.